examples/nextjs-cldogimage/components/Footer.jsx

```diff
@@ -0,0 +1,37 @@
+export default function Footer() {
+  return (
+    <div className="border-t pt-8">
+      <h2 className="mb-3 text-2xl font-bold">Resources</h2>
+      <p className="mb-2">
+        <a
+          href="https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/nextjs-cldogimage"
```

Comment:
if you end up updating the header with props, you could do the same thing here, again, just so that you can configure everything once in the layout 🤷  but nbd

Reply:
You mean change the link to go the app folder? If the app router demo is shown?

Reply:
just generally since you have to update this in every demo, that way the footer component is the same for you to copy and paste, and you set it once in the layout file. but i think having this link to the main example makes sense as you have it